apps/src/sites/studio/pages/maker/discountcode.js

```diff
@@ -4,15 +4,26 @@ import ReactDOM from 'react-dom';
 import EligibilityChecklist from '@cdo/apps/templates/EligibilityChecklist';
 import { Status } from '@cdo/apps/lib/kits/maker/ui/SetupStep';
 import getScriptData from '@cdo/apps/util/getScriptData';
+import DiscountCodeInstructions from '@cdo/apps/lib/kits/maker/ui/DiscountCodeInstructions';
 
 $(document).ready(() => {
   const scriptData = getScriptData('discountcode');
 
   ReactDOM.render(
-    <EligibilityChecklist
-      statusPD={scriptData.eligibility_status.statusPD ? Status.SUCCEEDED : Status.FAILED}
-      statusStudentCount={scriptData.eligibility_status.statusStudentCount ? Status.SUCCEEDED : Status.FAILED}
-    />
+    <div>
+      {false &&
+        <EligibilityChecklist
+          statusPD={scriptData.eligibility_status.statusPD ? Status.SUCCEEDED : Status.FAILED}
+          statusStudentCount={scriptData.eligibility_status.statusStudentCount ? Status.SUCCEEDED : Status.FAILED}
+        />
+      }
+      {true &&
+        <DiscountCodeInstructions
+          discountCode="XXX3XXX"
+          discountComplete={true}
+        />
+      }
+    </div>
```

Comment:
https://github.com/code-dot-org/code-dot-org/pull/18943 In my upcoming PR, I'm building in a mechanism whereby we can fake the scriptData by setting sessionStorage state, which will allow us to test this more easily (your change and mine will probably have some fairly easy to resolve merge conflicts).

In the meantime, I'd probably at least create a variable rather than hard coding true and false (though that variable might still be hard-coded to one or the other).